production/helm/loki/Chart.yaml

```
@@ -3,7 +3,7 @@ name: loki
 description: Helm chart for Grafana Loki in simple, scalable mode
 type: application
 appVersion: 2.6.1
-version: 3.0.3
+version: 3.0.4
```

```suggestion
version: 3.0.5
```
I've just noticed that it has been bumped before.